packages/blueprints-integration/src/content.ts

```diff
@@ -78,6 +80,13 @@ export interface RemoteContent extends BaseContent {
 	switcherInput: number | string
 }
 
+export interface EvsContent extends BaseContent {
+	studioLabel: string
+	switcherInput: number | string
+	evsChannel: string
+	uiColorCode?: string
```

Comment:
I don't think I like `uiColorCode`, because what is that code? I think it would be safer to just have this a color RGB in hash-HEX (#xxxxxx) format, same way as we do in `IBlueprintPart.invalidReason.color`: https://github.com/nrkno/tv-automation-server-core/blob/152f988e5692669516ead1ef8fb22ee395a69398/packages/blueprints-integration/src/rundown.ts#L145-L161

This way, we use the same color system for all places. I guess what you wanted was to be able to just say `red` here, but I think it's entirely possible, that some other system (not invented yet, but still) might need to either also display this, and it would not have the X11 Color Names table or would have to do some processing on it (say dim it, or add some transparency), in which case it would make it unnecessarily complicated.

Reply:
Makes sense, I guess representing it on some panel with colored LED buttons is not a too far-fetched example of something we'd like to be able to do. We don't validate the input anywhere though, so if we want some kind of security that it will always be a hex color code that should probably be added?

The reason for it to be named uiColor instead of simply color is that since the EVS channel names are colors (in wording, ex "GUL" (Norwegian for yellow)) I'd like for the property name to be unambiguous. So I'd like to keep the "ui" in the name for that reason.